src/core/dpl-cms-api/useProxyUrlGET.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  useQuery,
+  UseQueryOptions,
+  QueryFunction,
+  UseQueryResult,
+  QueryKey
+} from "react-query";
+import { getProxyUrlGETQueryKey, proxyUrlGET } from "./dpl";
+import type { ProxyUrlGETParams } from "./model";
+import { ErrorType } from "./mutator/fetcher";
+
+const useProxyUrlGET = <
```

Comment:
Consider if this warrants some documentation.

So this is our own alternative implementation of `useProxyUrlGET`. The reason why we need it here (support for `enabled`?) is not clear and others may assume that they can just use the generated hooks out of the box.

It would be nice with some explanation of why it is needed. If we cannot use the React Query hooks at all for DPL CMS then we should consider if/how it can be disabled for this API in Orval.

Reply:
After long investigation I found a way to avoid replicating the hook.
The `requestOptions: false` setting in the orval configuration prevents the `enabled` option to be overwritten.